workflow/workflow_commands.go

```diff
@@ -238,28 +239,32 @@ func UnmarshalMemoFromCLI(c *cli.Context) (map[string]interface{}, error) {
 	return memo, nil
 }
 
-type historyIterator struct {
-	iter interface {
-		HasNext() bool
-		Next() (*historypb.HistoryEvent, error)
-	}
+// historyTableIter adapts history iterator for Table output view
+type historyTableIter struct {
+	iter           iterator.Iterator[*historypb.HistoryEvent]
```

Comment:
What's this test failure about?

Reply:
requires this change https://github.com/temporalio/tctl-kit/pull/39